scripts/validationAndPreparation.sh

```diff
@@ -75,5 +76,7 @@ fi
 #safe new variables for next execution
 export TRUSTED_SETUP_TURN=$((TRUSTED_SETUP_TURN + 1))
```

Comment:
Are you sure that environment variables will persist across container restarts? I imagine the kubernetes job will restart from time to time and we don't want to verify the entire history again.

Reply:
If we are starting a new container, we have to inject the new  env TRUSTED_SETUP_TURN

Do you have a better solution/suggestion?